python/lsst/display/firefly/firefly.py

```diff
@@ -78,8 +79,8 @@ def __handleCallbacks(event):
         if data.get('type') == "POINT":
             lsst.log.debug("Event Received: %s" % data.get('id'))
 
-    def __init__(self, display, verbose=False, host="localhost", port=8080,
-                 name="afw", basedir="firefly", *args, **kwargs):
+    def __init__(self, display, verbose=False, host="localhost", port=8080, name="afw",
+                 basedir="firefly", *args, **kwargs):
```

Comment:
is name the channel name?  if so  we need to think of how to not this conflict.  Such as hostname + 'afw' or something like that.  If firefly is sharing these on the same server they would start updating each other.